Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.5
-ADD requirements.txt /tmp/requirements.txt
-RUN pip install -r /tmp/requirements.txt
+RUN pip install --upgrade pip && \
+    pip install Flask==0.10.1 pykube==0.14.0
```

Comment:
It would be better not to duplicate `pykube==x.y.z` instructions in two locations, in both `setup.py` and `Dockerfile`.  
You can use something like:

```
WORKDIR /code
ADD . /code
RUN pip install -e .[all]
```

in `Dockerfile`. In this manner, the packgaes dependencies are defined in one location only (`setup.py`).